src/screens/ProfileScreen.js

```diff
@@ -1,37 +1,29 @@
 import React from "react";
-import {
-  StyleSheet,
-  View
-} from "react-native";
+import { StyleSheet, View } from "react-native";
 
 import { useDispatch, useSelector } from "react-redux";
 import { setUserPhone } from "Store/actions";
 
-import { UserPhoneInput } from "Components/user-phone-input"
+import UserPhoneInput from "Components/user-phone-input";
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: "#fff",
-  },
+    backgroundColor: "#fff"
```

Comment:
please leave trailing commas for all lists

Reply:
Sorry, I think that's my code formatter. I'll review the settings.